packages/amplify-graphql-auth-transformer/src/graphql-auth-transformer.ts

```diff
@@ -261,6 +261,9 @@ Static group authorization should perform as expected.`,
           case QueryFieldType.LIST:
             this.protectListResolver(context, def, query.typeName, query.fieldName, acm);
             break;
+          case QueryFieldType.SYNC:
+            // protect sync query
```

Comment:
Should this be calling `protectSyncResolver()`?

Reply:
missed this line on the git add will add in the next commit